feedback-widget-cdk/README.md

```diff
@@ -0,0 +1,23 @@
+# Feedback Widget Infrastructure (AWS CDK - TypeScript)
+
+This project defines the infrastructure for the **Feedback Widget** using [AWS CDK](https://docs.aws.amazon.com/cdk/latest/guide/home.html) with TypeScript. It provisions an **Aurora PostgreSQL Serverless v2** database (DB) along with its supporting resources.
+
+## Resources
+
+| Resource              | Logical ID           | Purpose |
+|-----------------------|----------------------|---------|
+| **VPC**               | `AuroraVpc`          | Isolated network environment for the DB |
+| **Subnet Group**      | `AuroraSubnetGroup`  | Specifies DB placement in private subnets |
+| **Security Group**    | `AuroraSecurityGroup`| Restricts DB access to services inside VPC |
+| **Aurora Cluster**    | `AuroraCluster`      | Serverless v2 Aurora PostgreSQL DB |
+| **Outputs**           | `AuroraClusterEndpoint` | Cluster endpoint for local development or reference |
+
```

Comment:
[dust] For the most part, some of these assets are assumed to be Aurora-related, so having `AuroraCluster` be the name of your database cluster can be seen as redundant, no?

Maybe we can replace some of these resource names with a description of the thing being held, like `FeedbackDBCluster` or something like that? What do you think?